decorator.py

```diff
@@ -0,0 +1,28 @@
+import time
+from functools import wraps
+
+def get_time():
+    return time.time()
+
+
+def time_slow(threshold):
+    def wrapped_func(f):
+        @wraps(f)
+        def wrapper(*args, **kwargs):
+            start = get_time()
+            result = f(*args, **kwargs)
+            stop = get_time()
+            if threshold and stop - start > threshold:
```

Comment:
Since we are reusing stop - start value in 3 places wouldn't it make sense to extract it into a variable and not do the subtract every time?